src/app/app.module.ts

```diff
@@ -17,6 +18,8 @@ import { AppComponent } from "./app.component";
         BrowserModule,
         BrowserAnimationsModule,
         AppRoutingModule,
+        FormsModule,
+        ReactiveFormsModule,
```

Comment:
This was the only place I could import these two modules without getting a null injector error. I don't know why the imports/exports in the SharedModule aren't getting used.

Reply:
weird lets leave it for later